core/encoding.go

```diff
@@ -83,7 +83,7 @@ func GetEncodingParams(minChunkLength, minNumChunks uint) (EncodingParams, error
 // ValidateEncodingParams takes in the encoding parameters and returns an error if they are invalid.
 func ValidateEncodingParams(params EncodingParams, blobLength, SRSOrder int) error {
 
-	if int(params.ChunkLength*params.NumChunks) >= SRSOrder {
+	if int(params.ChunkLength*params.NumChunks) > SRSOrder {
```

Comment:
oh, because [s^i] corresponding to term x^i (in p(x))?

Reply:
I had to go back and look at why the constraint existed to begin with. See the justification here: 

https://github.com/Layr-Labs/eigenda/pull/247#issuecomment-1934528171